### ISSUE TITLE
Address the Google Manifesto

### ISSUE THREAD

**Issue by thejessleigh** [sublimemarch/so-you-just-learned#22]
We definitely want to come up with some good resources to link to here. However, we feel like we need more space and we want to link to some constructive conversations coming out in the aftermath, and give the Google Manifesto itself as little airtime as possible. It's had plenty.

**Comment by sublimemarch** [comment]
I definitely agree that we should give ourselves a week or two of space before we find a way to add this in, especially because the story is still developing.

**Comment by carlydotwebsite** [comment]
just reminded by _someone_ making the news again that we should still do this; I don't know if there's been a good roundup somewhere to link to since the initiating event

**Comment by sublimemarch** [comment]
Alright, I'm taking a look at articles and thinking how best to address this. Do we want to just add a summary of what happened? Or an article that shows how this is part of a broader problem? Or an article doing a line-by-line critique of the manifesto? I'm personally not sure of the best way to deal with it.

A listicle with example quotes: https://www.cnbc.com/2017/08/07/shocking-quotes-from-the-viral-google-manifesto.html

Some Googlers supported the ideas in the manifesto: https://www.wired.com/story/internal-messages-james-damore-google-memo/

Google's response: https://motherboard.vice.com/en_us/article/vbv54d/google-on-anti-diversity-manifesto-employees-must-feel-safe-sharing-their-opinions

**Comment by carlydotwebsite** [comment]
I think probably something that shows how this is part of a broader problem and illustrates why it was this big of a deal is probably good for the tone of this repo? Although possibly this thing may be big enough to warrant more than one article (since I could see a summary of what happened also being useful to people learning about stuff after-the-fact)

**Comment by sublimemarch** [comment]
Now that we've given this one some time, I think it's safe to say that this merits inclusion. Whatcha think @thejessleigh @cmho ?

**Comment by thejessleigh** [comment]
Definitely. There's also been a lot of think pieces and chatter since then. Do we want to try to find a more up to date version, or stick with the original docs?
